yarn.lock

```diff
@@ -12791,6 +12791,13 @@ ovh-api-services@^9.39.0:
   dependencies:
     lodash "^4.17.15"
 
+ovh-api-services@^9.39.0:
```

Comment:
I think a rebase on develop can be made as latest in the monorepo is now `9.39.1` I think

Reply:
This will be taken care of when we merge feat/veeam-backup branch.

Reply:
This has been addressed in the `feat/veeam-backup` branch. Probably you should take this change from the base branch